PurrSoft/PurrSoft.Application/Commands/AnimalCommands/AnimalCommands.cs

```diff
@@ -0,0 +1,36 @@
+﻿using PurrSoft.Application.Common;
+using PurrSoft.Domain.Entities;
+using PurrSoft.Domain.Entities.Enums;
+
+namespace PurrSoft.Application.Commands.AnimalCommands;
+
+public class AnimalGetCommand: BaseRequest<CollectionResponse<Animal>>
+{
+
+}
+
+public class AnimalCreateCommand: BaseRequest<CommandResponse<int>>
```

Comment:
So you want technically to send an int for what? I do not know if I am int the wrong commit but for me the handlers are not implemented? Did you do that because you wanted to? Or did you forget to implement it?

Reply:
In many CRUD interfaces I've seen, the int represents the newly added Id.
I will keep that but I'll instead change it to Guid.